draft-ietf-quic-ack-frequency.md

```diff
@@ -323,10 +323,17 @@ acknowledgements.
 
 ## Expediting Congestion Signals {#congestion}
 
-As specified in Section 13.2.1 of {{QUIC-TRANSPORT}}, an endpoint SHOULD
-immediately acknowledge packets marked with the ECN Congestion Experienced (CE)
-codepoint in the IP header. Doing so reduces the peer's response time to
-congestion events.
+And endpoing SHOULD send an immediate acknowledgement when a packet marked
+with the ECN Congestion Experienced (CE) codepoint in the IP header is
+received after the previous packet was not marked CE.
+
+An endpoint SHOULD also send an immediate acknowledgement when at least
+max(2, Ack-Eliciting Threshold) packets marked with the ECN Congestion
+Experienced (CE) codepoint in the IP header are received. Doing so reduces the
+peer's response time to congestion events, while also reducing the ACK rate
+compared to Section 13.2.1 of {{QUIC-TRANSPORT}} when peers are using
```

Comment:
This line is a bit confusing. I think we can separate out `reduces the peer's response time to congestion events` and `while also reducing the ACK rate as compared to`. Another clarification is needed about the fact that congestion controllers don't mark but respond to marks.
Suggestion:
```
Doing so reduces the peer's response time to congestion events especially when peers are using DCTCP {{?RFC8257}} or other congestion controllers that respond to immediate CE markings which are much more
 frequent than classic ECN {{?RFC3168}}.
 ```
 And then talk about
 ```
 This will also help in reducing the ACK rate on the reverse path as compared to Section 13.2.1 of {{QUIC-TRANSPORT}}
 ```

Reply:
It doesn't actually reduce the response time compared to RFC9000, so I think the current text is more accurate?

Reply:
I copied the `reduces response time` phrase from the current text.
I agree that it doesn't reduce peer's response time to congestion necessarily but makes the response based on the extent of congestion (number of CE marks) instead of its presence. (RFC 9000)

Reply:
The PR had already been updated to be more similar to your text, but I added 'extreme congestion', PTAL